pyodide_build/cli/xbuildenv.py

```diff
@@ -205,3 +205,27 @@ def _search(
         print(MetadataView.to_json(views))
     else:
         print(MetadataView.to_table(views))
+
+
+@app.command("install-emscripten")
+def _install_emscripten(
+    version: str = typer.Option(
+        "latest", help="Emscripten SDK Version (default: latest)"
```

Comment:
The default should be the one in the xbuildenv. Pyodide requires the same Emscripten version that is used to build the Pyodide runtime.

You can call `get_build_flag("PYODIDE_EMSCRIPTEN_VERSION")` to get the proper version.